.travis.yml

```diff
@@ -39,83 +46,38 @@ matrix:
       env: BUILD="base" TEST_TARGET="scalding-hadoop-test"
       script: "scripts/run_test.sh"
 
-    - scala: 2.10.4
-      env: BUILD="base" TEST_TARGET="scalding-jdbc scalding-json"
-      script: "scripts/run_test.sh"
-
-    - scala: 2.11.5
-      env: BUILD="base" TEST_TARGET="scalding-jdbc scalding-json"
-      script: "scripts/run_test.sh"
-
-    - scala: 2.10.4
-      env: BUILD="base" TEST_TARGET="scalding-macros"
-      script: "scripts/run_test.sh"
-
-    - scala: 2.11.5
-      env: BUILD="base" TEST_TARGET="scalding-macros"
-      script: "scripts/run_test.sh"
-
-# not committed yet
-    # - scala: 2.10.4
-    #   env: BUILD="base" TEST_TARGET="scalding-commons-macros"
-    #   script: "scripts/run_test.sh"
-
-    # - scala: 2.11.5
-    #   env: BUILD="base" TEST_TARGET="scalding-commons-macros"
-    #   script: "scripts/run_test.sh"
-
-    - scala: 2.10.4
-      env: BUILD="base" TEST_TARGET="scalding-parquet scalding-parquet-scrooge"
-      script: "scripts/run_test.sh"
-
-    - scala: 2.11.5
-      env: BUILD="base" TEST_TARGET="scalding-parquet scalding-parquet-scrooge"
-      script: "scripts/run_test.sh"
-
-    - scala: 2.10.4
-      env: BUILD="base" TEST_TARGET="scalding-repl"
-      script: "scripts/run_test.sh"
-
-    - scala: 2.10.4
-      env: BUILD="test tutorials"
-      script:
-      - "scripts/build_assembly_no_test.sh scalding"
-      - "scripts/test_tutorials.sh"
-
-    - scala: 2.11.5
-      env: BUILD="test tutorials"
+    - scala: 2.10.5
+      env: BUILD="test tutorials and matrix tutorials and repl" TEST_TARGET="scalding-repl"
```

Comment:
what does this syntax mean? "test tutorials *and* .. *and*"?

What is consuming the BUILD variable?